apps/dashboard/app/javascript/dynamic_forms.js

```diff
@@ -418,7 +418,6 @@ function updateVisibility(event, changeId) {
   $(`#${changeId}`).parents().each(function(_i, parent) {
     if(parent.classList.contains('form-group')) {
       changeElement = $(parent);
-      return false;
```

Comment:
The problem here is that `text_field` is being passed a `label` option, which wraps the label/input HTML combination in its own `div.form-group` element. So far simply searching for the top `.form-group` parent has been easier than over-riding the `wrapper_class` on the `text_field`. ... But this isn't my favorite option. I'd rather figure out how to ensure that each widget is only wrapped in one `.form-group` div.

Reply:
Which is to say that special cases may just be the best thing we can do right now.